components/Home.js

```diff
@@ -2,13 +2,15 @@ import { View, StyleSheet } from 'react-native'
 import Header from './Header'
 import GreetingButton from './GreetingButton'
 import Carousel from './Carousel/Carousel'
+import BottomNavBar from './BottomNavBar'
 
 const Home = () => {
     return (
         <View style={styles.homeContainer}>
             <Header />
             <GreetingButton />
-            <Carousel />
```

Comment:
I wonder why the nav bar is disappearing when the carousel is visible.

Reply:
Maybe the navbar needs to live inside the App rather than the home page, as it will need to persist throughout the application, not just on the home page?